config/routes.rb

```diff
@@ -4,11 +4,14 @@
 
   devise_for :users
 
-  resources :users, only: [:index, :show]
+  resources :users, only: [:index, :show] 
+  resources :friendships, only: [:create, :destroy]
+  
   resources :posts, only: [:index, :create] do
     resources :comments, only: [:create]
     resources :likes, only: [:create, :destroy]
   end
+  post 'accept', to: 'friendships#accept'
 
```

Comment:
### OPTIONAL
- [ ] **a better approach will be calling  it with  namespace or path, check [this docs](https://guides.rubyonrails.org/routing.html#overriding-the-new-and-edit-segments)**